elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ElytronOidcSubsystemTransformers.java

```diff
@@ -9,7 +9,20 @@
 import static org.wildfly.extension.elytron.oidc.ElytronOidcClientSubsystemModel.VERSION_2_0_0;
 import static org.wildfly.extension.elytron.oidc.ElytronOidcClientSubsystemModel.VERSION_3_0_0;
 import static org.wildfly.extension.elytron.oidc.ElytronOidcClientSubsystemModel.VERSION_4_0_0;
-import static org.wildfly.extension.elytron.oidc.ElytronOidcDescriptionConstants.*;
+import static org.wildfly.extension.elytron.oidc.ElytronOidcDescriptionConstants.AUTHENTICATION_REQUEST_FORMAT;
```

Comment:
The subsystem bump commit should just require the changes for the bump itself (i.e., the typical changes to increment the version but not the actual changes for the RFE. The RFE related changes should be part of the other commit).

Reply:
Just a reminder about this one for when you're back.

Reply:
Done!